examples/gnrc_minimal/Makefile

```diff
@@ -7,10 +7,10 @@ BOARD ?= native
 # This has to be the absolute path to the RIOT base directory:
 RIOTBASE ?= $(CURDIR)/../..
 
-# Comment this out to disable code in RIOT that does safety checking
+# Set this to 1 enable code in RIOT that does safety checking
```

Comment:
`to enable`? (: otherwise, the idea of further stripping down the minimal example is nice ..